bin/start

```diff
@@ -54,18 +58,22 @@ done
 
 export RUN_IN_DOCKER
 export CONJUR_DEPLOYMENT
-
-source test/bootstrap.env
+export DEV_ENV
 
 # summon environment variable
 export CONJUR_MAJOR_VERSION=4
 
+source bootstrap.env
 cd test
+
 if [[ $DEMO = true ]]; then
   runScriptWithSummon ./run_demo.sh
 elif [[ $RUN_IN_DOCKER = true ]]; then
   # this option cannot run locally due to lack of docker credentials. It works only in the Jenkins pipeline
   runScriptWithSummon ./test_in_docker.sh
+elif [[ $DEV_ENV = true ]]; then
+  cd ../dev
+  runScriptWithSummon ./run_local.sh
```

Comment:
`runScriptWithSummon` is a function. what you mean is to write `runScriptWithSummon ../dev/run_local.sh`.

Reply:
considered that a for a while and invested some time in updated the paths. I don't want to touch changing directories for now because it is a wider effort of updating all the paths in the program because depending on which environment, it drops you in that directory and  cascades from there. Might consider doing this change in a separate commit